From 196b297e1bd3d05a6852d866e2ffab91bbd744d0 Mon Sep 17 00:00:00 2001
From: El Dainosor <eldainosor@gmail.com>
Date: Tue, 24 Sep 2019 16:54:42 -0300
Subject: [PATCH 1/1] devices: Add a experimental build support

This is now a thing due to Q builds starting to roll out soon.
For teasing purposes, let's attach a screenshot of the first btlg Q build for now: https://i.imgur.com/kzubwar.png
---
 _devicesbeta/surnia.md |  8 ++++++++
 devices-beta.json      | 20 ++++++++++++++++++++
 2 files changed, 28 insertions(+)
 create mode 100644 _devicesbeta/surnia.md
 create mode 100644 devices-beta.json

diff --git a/_devicesbeta/surnia.md b/_devicesbeta/surnia.md
new file mode 100644
index 0000000..b4b04ec
--- /dev/null
+++ b/_devicesbeta/surnia.md
@@ -0,0 +1,8 @@
+---
+codename: surnia
+fullname: Moto E 2015 (LTE)
+maintainer: ElDainosor
+filename: BootleggersROM-Queue4surnia.4.7-MadStinky-Shishufied-20190924-161348.zip
+newformat: true
+buildsize: 384928123
+---
diff --git a/devices-beta.json b/devices-beta.json
new file mode 100644
index 0000000..520a7b3
--- /dev/null
+++ b/devices-beta.json
@@ -0,0 +1,20 @@
+---
+permalink: /api/devices-beta
+---
+{
+{% for device in site.devicesbeta %}
+{% assign buildDateStart = device.filename.size | minus: 12 %}{% assign buildDateStartWithTime = device.filename.size | minus: 19 %}{% assign buildDateFinal = device.filename.size | minus: 4 %}"{{device.codename}}": {
+	"fullname": "{{device.fullname}}",
+	"maintainer": "{{device.maintainer}}",
+	"xdathread": "{{device.xdathread}}",
+	"filename": "{{device.filename}}",
+	{% if device.newformat %}"buildate": "{{device.filename | slice: buildDateStartWithTime,buildDateFinal | remove: ".zip"}}",{% else %}"buildate": "{{device.filename | slice: buildDateStart,buildDateFinal | remove: ".zip"}}-191140",{% endif %}
+	"buildsize": "{{device.buildsize}}",
+	"id": "{{ forloop.index }}",
+	"download": {% if device.directmirror %}"{{ device.directmirror }}/{{ device.filename }}",{% else %}"https://downloads.sourceforge.net/project/bootleggersrom/builds/{{ device.codename }}/{{ device.filename }}",{% endif %}
+	"downloadfolder": "https://sourceforge.net/projects/bootleggersrom/files/builds/{{ device.codename }}",
+	"mirrorlink": "{{device.mirrorlink}}",
+	"changelog": "{{device.changelog}}"
+}{% unless forloop.last %},{% endunless %}
+{% endfor %}
+}
-- 
2.21.0.windows.1

